config/config.go

```diff
@@ -0,0 +1,29 @@
+package config
+
+import (
```

Comment:
By convention, if you are importing only one library, you should do it in a single line
```go
import "github.com/spf13/viper"
```

Reply:
You're right. Thanks!